src/pages/OrganisationPage.vue

```diff
@@ -109,7 +128,10 @@
               />
             </div>
           </div>
-          <CaptionedLoadingSpinner v-if="loading" class="pb-3"> Loading Reports... </CaptionedLoadingSpinner>
+          <CaptionedLoadingSpinner v-if="loading && !errorMessage" class="pb-3">
+            Loading Reports...
+          </CaptionedLoadingSpinner>
+          <AppAlert v-else-if="errorMessage">No data</AppAlert>
```

Comment:
Give it some padding e.g `p-4` so that it's not kissing the walls. Doesn't look good. You'll likely need to wrap it in a div for that.

```suggestion
          <div v-else-if="errorMessage" class="p-4">
            <AppAlert>No data</AppAlert>
          </div>
```